### PR TITLE
VIH-7379 Hide IM for judge on private consultation

### PR DESCRIPTION
**Pull request by jdssunil**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7379

### Change description ###
Hide IM button for Judge on private consultation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html [hunk 1]
```diff
@@ -237,7 +237,7 @@ <h2 class="govuk-heading-m" *ngIf="isNotStarted()">Is everyone connected?</h2>
     </div>
 </div>
 
-<app-participant-chat *ngIf="isIMEnabled && !isAdminConsultation" [hearing]="hearing" [alwaysOn]="hearing.isInSession()"></app-participant-chat>
+<app-participant-chat *ngIf="defineIsIMEnabled() && !isAdminConsultation" [hearing]="hearing" [alwaysOn]="hearing.isInSession()"></app-participant-chat>
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html:240]: why does this work and using `isIMEnabled` doesnt?

do we still need `isIMEnabled`? it has no other uses, and looks like it doesnt work? if not could we remove, please :)

**Review by levibutler — CHANGES_REQUESTED** [review]
remove isIMEnabled

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html (hunk 1 above)
Comment (jdssunil) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html:240]: Removed redundant code

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts [hunk 2]
```diff
@@ -101,7 +100,7 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseComponent implemen
                     if (this.conference.audio_recording_required) {
                         this.initAudioRecordingInterval();
                     }
-                    this.isIMEnabled = this.defineIsIMEnabled();
+                    this.defineIsIMEnabled();
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts:103]: this line does nothing, its returning a boolean, not setting anything and not being used..
please could we remove it

Comment (jdssunil) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts]: I didn't notice this. Done :)

**Review by levibutler — APPROVED** [review]

**Comment by levibutler** [comment]
The ui has now changed, there will still be a button in the header bar with this solution but when the panel slides out the chat will be missing. Is this enough?

**Pull request merged**